### PR TITLE
Import redirects command

### PR DESCRIPTION
**Pull request by jpmartinspt**
Add a command that allows a user to import redirects from production.

# QA

- `dotrun`
- `dotrun exec flask database import-redirects-from-prod <your-assets-token>`
- `dotrun exec flask token create qa`
- go to `localhost:8017/redirects?token=<qa-token>`
- check we have redirects

### CODE REVIEW

**Comment by cristinadresch** [comment]
@goulinkh can you help with this pls

**Review by goulinkh — APPROVED** [review]
LGTM :+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Build stage: Install python dependencies
 # ===
-FROM ubuntu:focal AS python-dependencies
-RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+FROM ubuntu:bionic AS python-dependencies
```
Comment (goulinkh) [inline Dockerfile:3]: I was wondering why we changed from focal (20.04) to bionic (18.04) as it's a more up to date version?

Comment (jpmartinspt) [reply Dockerfile:3]: Pilbox pulls an old version of Pillow that fails to install under python3.8(focal).

**Pull request merged**